packages/core/src/amazonq/webview/ui/quickActions/handler.ts

```diff
@@ -126,13 +127,13 @@ export class QuickActionHandler {
         // if there is no scan tab, open a new one
         const currentTabType = this.tabsStorage.getTab(affectedTabId)?.type
         if (currentTabType !== 'unknown' && currentTabType !== 'welcome') {
-            affectedTabId = this.mynahUI.updateStore('', {
+            affectedTabId = this.mynahUI?.updateStore('', {
                 loadingChat: true,
             })
         }
 
         if (affectedTabId === undefined) {
-            this.mynahUI.notify({
+            this.mynahUI?.notify({
```

Comment:
should we do a `if (!this.mynahUI) return` at the top of this function? seems strange to have lots of logic running that repeatedly handles this condition .